src/main.ts

```diff
@@ -90,7 +95,7 @@ class Mode {
     const folder = path.dirname(confPath);
     const name = path.basename(folder);
     const conf = YAML.parse(confRaw);
-    let ints: {}[] = [];
+    const ints: {}[] = [];
```

Comment:
@noverby There are plenty of these changes around: ESlint suggested them. Sounds a little counterintuitive to be able to .push  to a const, but if the linter says so...

Reply:
Linter also suggests using: 
```suggestion
    const ints: Record<string, unknown>[] = [];
```

Linter suggest this on all {} across the file...

Reply:
Please don't change the {}[] to Records. Ideally we should just use the interfaces.

Reply:
I've followed the suggestions